src/App.jsx

```diff
@@ -14,45 +17,102 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FIELD_BY_LETTERS = 'letters';
+const SORT_FIELD_BY_LENGTH = 'length';
+const IS_REVESER_DEFAULT_VALUE = false;
```

Comment:
IS_REVESER_DEFAULT_VALUE is an unnecessary variable, you don't need it, just set the initial value as `false` to `useState`